<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="addFirmButton">
          <property name="text">
           <string>Dodaj firmu</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addIncomeButton">
          <property name="text">
           <string>Dodaj prihod</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addPayedButton">
          <property name="text">
           <string>Dodaj placanja</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Unesi firmu:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="firmLineEdit"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>392</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="addFrimToDb">
          <property name="text">
           <string>Dodaj firmu u bazu</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Firma:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidgetIncome"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Pazareno:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="incomeLine"/>
        </item>
        <item>
         <widget class="QCalendarWidget" name="calendarIncome"/>
        </item>
        <item>
         <widget class="QPushButton" name="importIncomeButton">
          <property name="text">
           <string>Unesi prihod u bazu</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Firma:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidgetPayed"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Uplaceno:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="payedLine"/>
        </item>
        <item>
         <widget class="QCalendarWidget" name="calendarPayed"/>
        </item>
        <item>
         <widget class="QPushButton" name="importPayedButton">
          <property name="text">
           <string>Unesi uplatu u bazu</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>639</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
